CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v4.0.0
+
+* **BREAKING CHANGES**
+  * Added a `listeners` variabe allowing developers to override default settings but will cause all.
```

Comment:
`variable`

Reply:
"but will cause all." trails off, could probably delete this bit since you explain it in the next line

Reply:
This line could probably be moved into a `New Features` section, then have the specific bits that break living on their own